### PR TITLE
feat(toolbox/native): custom overflow menu buttons

### PR DESCRIPTION
**Pull request by Calinteodor**

### CODE REVIEW

File: react/features/toolbox/components/native/OverflowMenu.tsx [hunk 1]
```diff
@@ -140,6 +159,23 @@ class OverflowMenu extends PureComponent<IProps, IState> {
             }
         };
 
+        const buttons = [ {
```
Comment (saghul) [inline react/features/toolbox/components/native/OverflowMenu.tsx:162]: Why is this change necessary?

File: react/features/toolbox/components/native/OverflowMenu.tsx [hunk 2]
```diff
@@ -170,6 +206,15 @@ class OverflowMenu extends PureComponent<IProps, IState> {
                 <ClosedCaptionButton { ...buttonProps } />
                 <SharedDocumentButton { ...buttonProps } />
                 <SettingsButton { ...buttonProps } />
+                <Divider style = { styles.divider as ViewStyle } />
+                {
+                    buttons.map(({ key,...rest }) => (
+                        <CustomOptionButton
```
Comment (saghul) [inline react/features/toolbox/components/native/OverflowMenu.tsx:212]: I'd put the custom button first on the list. of buttons.

File: react/features/toolbox/components/native/OverflowMenu.tsx (hunk 1 above)
Comment (Calinteodor) [reply react/features/toolbox/components/native/OverflowMenu.tsx:162]: This is just for testing. To see that UI wise, everything looks as expected. 
It will be removed and instead we will use const { customToolbarButtons } = state['features/base/config'];

Comment (saghul) [reply react/features/toolbox/components/native/OverflowMenu.tsx:162]: Ah okidoki!

**Review by saghul — DISMISSED** [review]

**Review by saghul — DISMISSED** [review]
Do data URLs with a base64 encoded PNG work?

**Review by saghul — APPROVED** [review]
